src/sentry/reprocessing2.py

```diff
@@ -9,17 +9,27 @@
 
 from sentry import nodestore, features, eventstore
 from sentry.attachments import CachedAttachment, attachment_cache
-from sentry.models import EventAttachment
+from sentry import models
 from sentry.utils import snuba
 from sentry.utils.cache import cache_key_for_event
 from sentry.utils.redis import redis_clusters
 from sentry.eventstore.processing import event_processing_store
+from sentry.deletions.defaults.group import GROUP_RELATED_MODELS
 
 logger = logging.getLogger("sentry.reprocessing")
 
 _REDIS_SYNC_TTL = 3600
 
 
+GROUP_MODELS_TO_MIGRATE = GROUP_RELATED_MODELS + (models.Activity,)
```

Comment:
After https://github.com/getsentry/sentry/pull/21611 this addition should not be necessary